cli/util.go

```diff
@@ -181,6 +182,49 @@ func isDigit(s string) bool {
 	}) == -1
 }
 
+// extendedParseDuration is a more lenient version of parseDuration that allows
+// for more flexible input formats and cumulative durations.
+// It allows for some extra units:
+//   - d (days)
+//   - y (years)
+func extendedParseDuration(raw string) (time.Duration, error) {
+	var duration time.Duration
+
+	// Regular expression to match any characters that do not match the expected duration format
+	invalidCharRe := regexp.MustCompile(`[^0-9|nsuµhdym]+`)
+	if invalidCharRe.MatchString(raw) {
+		return 0, xerrors.Errorf("invalid duration format: %q", raw)
+	}
+
+	// Regular expression to match numbers followed by 'd', 'y', or time units
```

Comment:
If you look at tests for the original duration,  you will find comma-separated [examples](https://github.com/golang/go/blob/b521ebb55a9b26c8824b219376c7f91f7cda6ec2/src/time/time_test.go#L612-L627).

I don't believe we have to support them immediately, but would great to add a note/FIXME in the code. Perhaps verify what happens if somebody defines `5d7.5h0.5m`.

Reply:
Just added a FIXME 😢  Nice one thanks for raising it.